README.md

```diff
@@ -1,14 +1,15 @@
 # python-libprio
```

Comment:
Maybe we could rename this repository to `prio-etl`?

Reply:
I'd rather wait on renaming the repository. I don't think ETL captures the nature of what's happening here, since the flow is more involved than a single extract, transform, load step into a data-store.